employee-rostering-gwtui/src/main/resources/org/optaweb/employeerostering/gwtui/client/admin/AdminPage.html

```diff
@@ -1,5 +1,32 @@
 <div>
     <h1>Admin</h1>
-    <button data-field="add-tenant-button">Add Tenant</button>
+    
+      <div class="row toolbar-pf">
+
+        <div class="col-sm-12">
+          <div class="toolbar-pf-actions search-pf kie-search-pf has-button">
+            <div data-field="search-bar"></div>
+
+            <div class="toolbar-pf-action-right">
+              <div data-field="pager"></div>
+              <div class="form-group">
+                <button data-field="refresh-button" type="button" class="btn btn-default">Refresh</button>
+                <button data-field="add-tenant-button">Add Tenant</button>
+              </div>
+            </div>
+          </div>
+        </div><!-- /col -->
+      </div>
+    <table data-field="table" class="table table-hover kie-tablelist">
+      <thead>
+        <tr>
+          <!-- TODO: i18n -->
```

Comment:
I suspect that these TODO: i18n comments should be just about everywhere in UI, which will just clutter the code.
Let's not keep copy pasting them to some area - and leave them out.

I'l also make a i18n issue in jira, so we can make work of putting all UI strings in i18n property files.

Reply:
Wait, a sec, we have i18n support thanks to your work: https://github.com/kiegroup/optaweb-employee-rostering/pull/13/files

Ok, in that case, fix this to use that i18n todo :)